default/stringtables/en.txt

```diff
@@ -6197,6 +6202,18 @@ SP_EXOBOT_DESC
 '''While not as efficient as fully self-directed citizens, Exobots can colonize otherwise inhospitable planets, and build more Exobots.
 '''
 
+SP_ANCIENT_GUARDIANS
+Ancient Guardians
+SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC
+'''Guardian Robots constructed by a Precursor civilization aeons ago
+[[encyclopedia ROBOTIC_SPECIES_TITLE]]
+Will self-destruct when conquered
+'''
+
+SP_ANCIENT_GUARDIANS_DESC
+'''Constructed by precusor technology these ancient machines can adapt to and fight in any kind of environment. With unwavering loyalty and commitment they have protected the planet from any invaders, never wasting time to repair and improve the fortifications. Upon being captured every Guardian Robot will automatically self-destruct.
```

Comment:
I am, but am terrible with punctuation, I checked with the editor-I-live-with and she says yes in all three, the latter two being optional for reading-as-text but required if to be read aloud. The first definitely as it's two clauses.

I wouldn't have put any of them in either, so good catch.